src/shutdown.cc

```diff
@@ -257,9 +259,15 @@ int main(int argc, char **argv)
     auto shutdown_type = shutdown_type_t::POWEROFF;
 
     const char *execname = base_name(argv[0]);
-    if (strcmp(execname, reboot_execname) == 0) {
+    if (strcmp(execname, halt_execname) == 0) {
+        shutdown_type = shutdown_type_t::HALT;
+    }
+    else if (strcmp(execname, reboot_execname) == 0) {
```

Comment:
From the manual page:

       The default action is to power down the system if called as either
       "shutdown", "halt", or "poweroff"

This breaks that.

Reply:
It's confusing for me that `halt` and `shutdown -h` have different effect but I'm  going to fix it to match the doc.

Reply:
The halt check is removed